app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadFragment.kt

```diff
@@ -56,6 +56,16 @@ class SellPayloadFragment :
         }
     }
 
+    override fun onStart() {
```

Comment:
heh =(

Reply:
?

Reply:
can we call presenter method to update data only on onResume?